### PR TITLE
K8SPXC-447: commit versionservice client

### PR DESCRIPTION
**Pull request by qJkee**
[K8SPXC-447](https://jira.percona.com/browse/K8SPXC-447)

Commit version service generated client

### CODE REVIEW

**Comment by blez** [comment]
I think we need to have a script or makefile target to regenerate code.

**Comment by qJkee** [comment]
> I think we need to have a script or makefile target to regenerate code.

But we have it)
`build/build-versionservice-client.sh` will regenerate code from swagger file

File: .gitignore [hunk 1]
```diff
@@ -79,5 +79,4 @@ client-secret.json
 e2e-tests/conf/cloud-secret.yml
 .DS_Store
 .idea/**
-e2e-tests/init-deploy/smallrun
-versionserviceclient/
+e2e-tests/init-deploy/smallrun
```
Comment (blez) [inline .gitignore:82]: It tells you need one more line at the end of the file

**Review by blez — DISMISSED** [review]

**Comment by blez** [comment]
> build/build-versionservice-client.sh will regenerate code from swagger file
We should mention it in Readme

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| scaling | passed |
| recreate | passed |
| haproxy | passed |
| storage | passed |
| big-data | passed |
| self-healing | passed |
| init-deploy | passed |
| scaling-proxysql | passed |
| upgrade | passed |
| restore-to-encrypted-cluster | passed |
| limits | passed |
| self-healing-advanced | passed |
| monitoring | passed |
| upgrade-consistency | passed |
| security-context | passed |
| operator-self-healing | passed |
| demand-backup | passed |
| monitoring-2-0 | passed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| scheduled-backup | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| tls-issue-self | passed |
| demand-backup-encrypted-with-tls | failed |
| tls-issue-cert-manager | passed |
| tls-issue-cert-manager-ref | passed |

File: .gitignore (hunk 1 above)
Comment (qJkee) [reply .gitignore]: 5cb5cd1

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| big-data | passed |
| storage | passed |
| scaling | passed |
| self-healing | passed |
| scaling-proxysql | passed |
| upgrade | passed |
| upgrade-consistency | passed |
| security-context | passed |
| self-healing-advanced | passed |
| operator-self-healing | passed |

**Review by delgod — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |

**Comment by JNKPercona** [comment]
PXC operator docker - perconalab/percona-xtradb-cluster-operator:PR-569-9e18842e

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| scaling | passed |
| haproxy | passed |
| storage | passed |
| big-data | passed |
| recreate | passed |
| init-deploy | passed |
| scaling-proxysql | passed |
| self-healing | passed |
| upgrade | passed |
| restore-to-encrypted-cluster | passed |
| limits | passed |
| upgrade-consistency | passed |
| monitoring | passed |
| self-healing-advanced | passed |
| security-context | passed |
| operator-self-healing | passed |
| demand-backup | passed |
| monitoring-2-0 | passed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| scheduled-backup | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| tls-issue-self | passed |
| demand-backup-encrypted-with-tls | passed |
| tls-issue-cert-manager | passed |
| tls-issue-cert-manager-ref | passed |

**Review by delgod — DISMISSED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| recreate | passed |
| storage | passed |
| scaling | passed |
| big-data | passed |
| haproxy | passed |
| scaling-proxysql | passed |
| self-healing | passed |
| init-deploy | passed |
| upgrade | passed |
| restore-to-encrypted-cluster | failed |
| self-healing-advanced | passed |
| limits | passed |
| upgrade-consistency | passed |
| security-context | failed |
| monitoring | passed |
| demand-backup | failed |
| operator-self-healing | passed |
| monitoring-2-0 | passed |
| affinity | passed |
| scheduled-backup | failed |
| one-pod | failed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| tls-issue-self | passed |
| tls-issue-cert-manager | passed |
| tls-issue-cert-manager-ref | passed |
| demand-backup-encrypted-with-tls | failed |

**Review by blez — APPROVED** [review]

**Pull request merged**